src/components/styles/BottomNavBar.styled.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components'
+
+export const StyledBottomNavBar = styled.div`
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  background-color: whitesmoke;
+  height: 8%;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`
+
+export const NavBarIcon = styled.div`
+  width: 25%;
+  margin: 10px;
+  text-align: center;
+`
+
+export const Icon = styled.img`
+  width: 30%;
+  height: 30%;
+`
+
+export const Avatar = styled.img`
```

Comment:
should 'avatar' be responsive?
![Screenshot 2021-10-10 at 11 40 10 PM](https://user-images.githubusercontent.com/57870728/136702970-90f6a99f-f5f1-4833-b8e2-ac4573a8993d.png)